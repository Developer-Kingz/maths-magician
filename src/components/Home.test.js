import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom'; // Import MemoryRouter
import Home from './Home';

test('Home component matches snapshot', () => {
  // Wrap Home component in MemoryRouter
  const { asFragment } = render(
    <MemoryRouter>
      <Home />
    </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
