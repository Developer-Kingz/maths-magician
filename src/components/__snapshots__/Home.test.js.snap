// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <div>
      <div
        class="navbar"
      >
        <div
          class="logo"
        >
          <h1>
            Math Magician
          </h1>
        </div>
        <div
          class="nav"
        >
          <ul>
            <a
              aria-current="page"
              class="active"
              href="/"
            >
              <li>
                Home
              </li>
            </a>
            <a
              class=""
              href="/calculator"
            >
              <li>
                Calculator
              </li>
            </a>
            <a
              class=""
              href="/quotes"
            >
              <li>
                Quotes
              </li>
            </a>
          </ul>
        </div>
      </div>
    </div>
    <h2>
      Welcome To Our Page
    </h2>
    <p
      class="para"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis accusamus accusantium sit fuga blanditiis illum corrupti, maxime consectetur eveniet quos, nulla fugiat culpa eum asperiores doloribus quasi iste porro similique id mollitia libero tenetur aliquam? Distinctio dignissimos assumenda ipsa quas earum dolorem rem a commodi eaque. Quibusdam esse quae ipsa!
    </p>
    <br />
    <p
      class="para"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum modi, ad velit fugit ducimus deserunt labore debitis, temporibus magnam pariatur rerum sequi! Placeat molestias ipsa eos eaque accusamus veniam. Corrupti aut nostrum, optio, aliquam placeat alias doloremque sit magni quo illum earum voluptatem autem, ut adipisci. Ipsam minima ad ipsum fugit molestiae reprehenderit harum. Illum, cumque aliquid? Modi, recusandae ab eligendi ullam, placeat consectetur magnam, dolorem accusamus earum tenetur nihil minima maiores labore. Voluptates nulla, ex eos quaerat eaque eveniet excepturi corporis, nemo repellendus numquam a possimus at non dolorum molestiae velit quam earum cumque rem laboriosam hic. Accusamus, in?
    </p>
    <br />
    <p
      class="para"
    >
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Doloribus dolorem autem praesentium dolores sunt, quibusdam libero adipisci, aperiam id incidunt, aliquam explicabo corporis ducimus sint aliquid nobis maiores nisi repudiandae enim! Tempore accusamus corrupti a quasi veritatis dolorum aspernatur veniam nobis velit? Qui labore, id consectetur, omnis laudantium vitae officia architecto sed quo perspiciatis ipsam libero aperiam praesentium! Labore tempore laboriosam itaque, provident esse dolores eius! Error, dolore. Quibusdam, sint libero illo id laudantium maiores quia dolore doloremque deleniti ratione voluptatibus molestias iure nobis enim ut officia omnis, earum culpa aspernatur. Adipisci, laboriosam placeat ut officia laborum aperiam facere blanditiis in illo veniam suscipit sapiente aliquam eaque non itaque corrupti, libero temporibus aut incidunt maiores porro, quasi quisquam. Maiores atque sequi culpa at recusandae, dolore numquam placeat, voluptates sunt adipisci ducimus rem voluptatibus, voluptate reiciendis in. Corrupti, quia, deserunt explicabo tenetur unde eveniet similique tempore sapiente eos recusandae, mollitia sit expedita quam! Inventore, mollitia facilis iste obcaecati velit cumque aspernatur dolor distinctio doloremque ex error rerum officiis repellendus nam accusantium, recusandae blanditiis fugit, labore harum laboriosam fugiat. Exercitationem a minima possimus, reiciendis temporibus accusantium repellendus deserunt odio error inventore culpa quisquam vero sit excepturi quaerat soluta alias nulla! Porro, sequi.
    </p>
  </div>
</DocumentFragment>
`;
